src/test/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static final String KEY = "apple";
+    private static final Integer DEFAULT_QUANTITY = 20;
+    private static OperationHandler operationHandler;
+    private static FruitTransaction fruitTransaction;
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void init() {
+        operationHandler = new PurchaseOperationHandler();
+        fruitDao = new FruitDaoImpl();
+        fruitTransaction = new FruitTransaction();
+    }
+
+    @Before
+    public void setUp() {
+        fruitTransaction.setFruit(KEY);
+        fruitTransaction.setOperation(FruitTransaction.Operation.PURCHASE);
```

Comment:
(Suggestion only) Can't you transfer these values right to the constructor on 25 line?